Signal/src/views/OWSAlerts.swift

```diff
@@ -23,15 +23,18 @@ import Foundation
         UIApplication.shared.frontmostViewController?.present(alertController, animated: true, completion: nil)
     }
 
+    public class func showAlert(withTitle title: String) {
+        self.showAlert(withTitle: title, message: nil, buttonTitle: NSLocalizedString("OK", comment: ""))
+    }
+
     public class func showAlert(withTitle title: String, message: String) {
```

Comment:
nit: default params might dedupe this helpfully

https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Functions.html#//apple_ref/doc/uid/TP40014097-CH10-ID169